tools/debugging/README.md

```diff
@@ -73,6 +73,17 @@ $ GST_DEBUG="GST_TRACER:7" GST_TRACERS="leaks(GstEvent,GstMessage)" \
 gst-launch-1.0 videotestsrc num-buffers=10 ! fakesink
 ```
 
+#### Case study 2: Tracing individual NNStreamer elements with GST_DEBUG and silent property
+
+Each NNStreamer element has the `silent` property. An NNStreamer element can be traced in verbose mode by setting `FALSE` to the `silent` property.
+
+```bash
+$ GST_DEBUG="tensor_decoder:7" \
+gst-launch-1.0 fakesrc ! tensor_decoder silent=FALSE ! fakesink 
```

Comment:
This is also an erroneous pipeline.
How about changing it like this?
`gst-launch-1.0 videotestsrc ! tensor_converter silent=false ! tensor_sink`

Reply:
Cool. Didn't know that tensor_converter wouldn't throw an error here. I changed it to the following code and it prints debugging messages very well :)
```
$ GST_DEBUG="tensor_converter:7" \
gst-launch-1.0 videotestsrc ! tensor_converter silent=false ! tensor_sink
```